weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx

```diff
@@ -278,9 +278,13 @@ export const ObjectViewer = ({apiRef, data, isExpanded}: ObjectViewerProps) => {
   // the expansion. Importantly, when the column is clicked, we do some
   // bookkeeping to add the expanded ref to the `expandedRefs` state. This
   // triggers a set of state updates to populate the expanded data.
-  const [expandedIds, setExpandedIds] = useState<Array<string | number>>(
-    isExpanded ? rows.filter(r => !r.isExpandableRef).map(r => r.id) : []
```

Comment:
this only applies on the very first mount, so on subsequent renders with `isExpanded` **now** set to `true`, because the data below is memoized, `setExpandedIds` remains empty.